### PR TITLE
fix(VRangeSlider): add unique id for inputs elements

### LINKED ISSUE #6843: [Bug Report] v-slider generates duplicate element id's
### Versions and Environment
**Vuetify:** 1.5.7
**Vue:** 2.6.10
**Browsers:** Chrome 73.0.3683.86
**OS:** Mac OS 10.14.3

### Steps to reproduce
1. Use a Vuetify v-slider component with a custom id
2. See the generated HTML (both the container div and the input elements have the same id)

### Expected Behavior
Each element should have a unique id, if both container and input elements need an id there could be a `container-id` similar to the suggested solution in [#2777](https://github.com/vuetifyjs/vuetify/issues/2777). Otherwise the id could be auto-generated with `container` appended or only the input should have the id. (probably best to stay consistent with other form controls)

```javascript
<v-slider
   id="slider-input-id"
   container-id="slider-container-id"
   v-model="slider"
></v-slider>
```

### Actual Behavior
```javascript
<v-slider
   id="slider-id"
   v-model="slider"
></v-slider>
```

Also see the HTML output on [this demo](https://codepen.io/robinvenneman/pen/RdmwOa)

### Reproduction Link
<a href="https://codepen.io/robinvenneman/pen/RdmwOa" target="_blank">https://codepen.io/robinvenneman/pen/RdmwOa</a>

### PR DESCRIPTION
**Pull request by VSymonenko**
## Description
Resolve #6843 

## Motivation and Context
Elements ids should be unique on a page

## How Has This Been Tested?
Created a unit test

## Markup:
No additional configuration required

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VRangeSlider/VRangeSlider.ts [hunk 1]
```diff
@@ -107,6 +107,7 @@ export default VSlider.extend({
         input.data = input.data || {}
         input.data.attrs = input.data.attrs || {}
         input.data.attrs.value = this.internalValue[i]
+        input.data.attrs.id = !i ? `min-${this._uid}` : `max-${this._uid}`
```
Comment (MajesticPotatoe) [inline packages/vuetify/src/components/VRangeSlider/VRangeSlider.ts]:
```suggestion
        input.data.attrs.id = !i ? `input-min-${this._uid}` : `input-max-${this._uid}`
```

File: packages/vuetify/src/components/VRangeSlider/__tests__/VRangeSlider.spec.ts [hunk 2]
```diff
@@ -220,4 +220,12 @@ describe('VRangeSlider.ts', () => {
       expect(wrapper.vm.internalValue).toEqual([0, 0])
     })
   })
+
+  // https://github.com/vuetifyjs/vuetify/issues/6843
+  it('should be uniq id', async () => {
+    const wrapper = mountFunction()
+
+    const [min, max] = wrapper.vm.genInput()
```
Comment (johnleider) [inline packages/vuetify/src/components/VRangeSlider/__tests__/VRangeSlider.spec.ts]: Move to line 227, keep line 228

File: packages/vuetify/src/components/VRangeSlider/VRangeSlider.ts [hunk 3]
```diff
@@ -107,6 +107,7 @@ export default VSlider.extend({
         input.data = input.data || {}
         input.data.attrs = input.data.attrs || {}
         input.data.attrs.value = this.internalValue[i]
+        input.data.attrs.id = !i ? `input-min-${this._uid}` : `input-max-${this._uid}`
```
Comment (johnleider) [inline packages/vuetify/src/components/VRangeSlider/VRangeSlider.ts]:
```suggestion
        input.data.attrs.id = `input-${!i ? 'min' : 'max'}-${this._uid}`
```

File: packages/vuetify/src/components/VRangeSlider/__tests__/VRangeSlider.spec.ts (hunk 2 above)
Comment (VSymonenko) [reply packages/vuetify/src/components/VRangeSlider/__tests__/VRangeSlider.spec.ts]: fixed at https://github.com/vuetifyjs/vuetify/pull/10654/commits/abf6a61de0e52da540c67b388936a38b0f3b771a

**Comment by VSymonenko** [comment]
changed logic as in the suggestion, see https://github.com/vuetifyjs/vuetify/pull/10654/commits/cfeb50b9e936e2d524a79a88c20f4731e0501c05

File: packages/vuetify/src/components/VRangeSlider/VRangeSlider.ts [hunk 4]
```diff
@@ -107,6 +107,7 @@ export default VSlider.extend({
         input.data = input.data || {}
         input.data.attrs = input.data.attrs || {}
         input.data.attrs.value = this.internalValue[i]
+        input.data.attrs.id = i ? `input-max-${this._uid}` : `input-min-${this._uid}`
```
Comment (johnleider) [inline packages/vuetify/src/components/VRangeSlider/VRangeSlider.ts]:
```suggestion
        input.data.attrs.id = `input-${i ? 'max' : 'min'}-${this._uid}`
```

**Comment by VSymonenko** [comment]
thanks all), sorry I didn't have time for requests

**Pull request merged**